src/components/MemberList.tsx

```diff
@@ -0,0 +1,33 @@
+import { useAuth0 } from "@auth0/auth0-react";
+import { IonItemSliding, IonItem, IonAvatar, IonImg, IonLabel, IonItemOptions, IonItemOption, IonIcon, IonNote } from "@ionic/react";
+import { closeOutline } from "ionicons/icons";
+
+interface MemberListProps {
+    users?: [] // should have all the users that are gonna appear
+}
+
+const MemberList: React.FC<MemberListProps> = ({
+    users
+}) => {
+
+    const { user } = useAuth0();
+
+    return (
+        <IonItemSliding>
```

Comment:
right now this is kind of static and i'm assuming is only displaying the user who is logged in. think we may need to add .map method where you generate ionavatar/ionimg/etc for each user in your array. you can take a look at how i did that in customlist for example.

Reply:
yeah so i was doing that originally and then i realized that i needed to figure out how to pass in multiple user objects at a time and idk how to do that as of rn so i've shelved that until later

Reply:
ahh i see. ok that makes sense, in that case we can push that task out